about.html

```diff
@@ -0,0 +1,221 @@
+            </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">English </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">My Page </a>
+          </li>
+          <li class="icon">
+            <a href="#" class="none">Logout </a>
+          </li>
+        </ul>
+      </div>
+
+      <nav class="nav-bar">
+        <a href="index.html" class="logo">Math Events</a>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="nav-menu">
+          <ul class="nav-links">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#" class="news">News</a></li>
+          </ul>
+          <button class="math" href="#">Math Events</button>
+        </div>
+      </nav>
+    </header>
+    <div class="welcome-about">
+      <div class="container6">
+        <div class="about">
+          <h4>73rd FIFA Congress Meeting</h4>
+          <h2>
+            CREATIVE COMMONS<br />
+            GLOBAL SUMMIT 2023
+          </h2>
+          <hr class="line4" />
+        </div>
+        <div class="welcome-about2">
+          <h1>"Hello!Sharing world"</h1>
+          <h2>
+            CREATIVE COMMONS<br />
+            GLOBAL SUMMIT 2023
+          </h2>
+        </div>
+
+        <div class="texts">
+          <p class="p3">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years.
+          </p>
+
+          <p class="p4">
+            This year, we hope to expand our invitation list including
+            organizations and individuals who want to work with us on shared
+            projects that advance the cause of the Commons, free culture and
+            open knowledge.
+          </p>
+          <p class="p5">
+            So if you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+        </div>
+      </div>
+
+      <div class="texts2">
+        <p class="p3">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Commons affiliate network in a
+          different country every two years. This year, we hope to expand our
+          invitation list including organizations and individuals.
+        </p>
+      </div>
```

Comment:
- [ ] 🙏Please adjust the size of this `div` on the mobile screen📱 to create space on the right and left sides, resembling the original design. This will enhance the appearance🌹 of your design and make it as cool as the original.

Your design:
![image](https://github.com/meryemsanem/capstone-project/assets/63445083/edf52405-ac72-413e-b279-caa304b8db0e)

Original design:
![image](https://github.com/meryemsanem/capstone-project/assets/63445083/02f09bbe-ed54-4362-bce5-10790004c123)